integrations/huggingface/src/index.ts

```diff
@@ -0,0 +1,115 @@
+import { interfaces } from "@botpress/sdk";
+import { HfInference } from "@huggingface/inference";
+import * as bp from ".botpress";
+import axios from "axios";
+import { generateContent } from "./actions/generateContent";
+
+export default new bp.Integration({
+  register: async ({ ctx, client, logger }) => {
+    if (!ctx.configuration.languageModels) {
+      return;
+    }
+
+    const modelIds = ctx.configuration.languageModels
+      .split(",")
+      .map((el) => el.trim());
+
+    if (modelIds.length > 10) {
+      logger.forBot().debug(`Too many models provided`);
+      return;
```

Comment:
why did you add this limit?

Reply:
Idk if there's a state limit, but if there is this will protect it

Reply:
@ermek-botpress its a good practice in general, lets bump it if needed. However, I think it would be better to throw a `RuntimeError` here so that the registering step fails louder